examples/spark/images/zeppelin-build/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Copyright 2015 The Kubernetes Authors All rights reserved.
```

Comment:
Probably want to denote somewhere that the whole purpose of this image is simply to produce the other image.